tools/model_tools/setup.py

```diff
@@ -58,7 +58,7 @@ class CustomBuild(distutils.command.build.build):
     def run(self):
         super().run()
 
-        package_build_dir = Path(self.build_lib, 'open_model_zoo/model_tools')
+        package_build_dir = Path(self.build_lib, 'openvino/model_zoo')
```

Comment:
Please add __init__.py under ` openvino` folder with content like shown in a link below to support [pkgutil-style namespace packages](https://packaging.python.org/guides/packaging-namespace-packages/#pkgutil-style-namespace-packages)

https://github.com/openvinotoolkit/openvino/blob/master/tools/benchmark_tool/openvino/__init__.py

Reply:
Done